u19_pipeline/recording.py

```diff
@@ -0,0 +1,181 @@
+import datajoint as dj
+from u19_pipeline import lab, task, subject
+
+schema = dj.schema('u19_recording')
+
+from u19_pipeline import acquisition, subject
+
+
+@schema
+class RecordingModality(dj.Lookup):
+     definition = """
+     recording_modality:   varchar(64)                  # modalities for recording (ephys, imaging, video_recording, etc.) 
+     ---
+     modality_description: varchar(255)                 # description for the modality
+     root_direcory:        varchar(255)                 # root directory where that modality is stored (e.g. ephys = /braininit/Data/eletrophysiology)
+     """
+     contents = [
+        ['ephys',             '', '/braininit/Data/eletrophysiology'],
+        ['imaging',           '', '/braininit/Data/eletrophysiology'],
+        ['video_acquisition', '', '/braininit/Data/video_acquisition']
```

Comment:
```suggestion
        ['ephys', 'Neuropixels', '/mnt/cup/braininit/Data/electrophysiology'],
        ['imaging', 'Two-photon calcium imaging', '/mnt/cup/braininit/Data/imaging'],
        ['video_acquisition', '', '/mnt/cup/braininit/Data/video_acquisition']
```

Reply:
I prepended `/mnt/cup`.  Is this required?

Reply:
well I was thinking just /braininit is ok to be a reference for the actual path.
/braininit is /mnt/cup/braininit in spock, scotty
/braininit is /Volumes/braininit on Mac
/braininit is \\cup.pni.princeton.edu\braininit\ on Windows

Reply:
Thanks for clarifying.  Sounds great.